src/pages/office/office.html

```diff
@@ -19,4 +19,36 @@
         <button ion-button navPush="TerminateTestPage">Terminate Test</button>
       </ion-card-content>
     </ion-card>
+
+    <ion-card id="ETA" *ngIf="(pageState.etaFaults$ | async)">
+        <ion-card-content>
+          <ion-grid>
+            <ion-row align-items-start class="mes-data">
+              <ion-col col-3></ion-col>
```

Comment:
Why is there an Empty ion-col?

Reply:
its to do with the spacing between the label and the fault, is there another way to space it without adding the column?

Reply:
Just add left padding to the `.fault-heading` class?

Reply:
fixed